recipes/extra-cmake-modules/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class ExtracmakemodulesConan(ConanFile):
+    name = "extra-cmake-modules"
+    license = ("MIT", "BSD-2-clause", "BSD-3-clause")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://api.kde.org/ecm/"
+    topics = ("conan", "cmake")
+    description = "KDE's CMake modules"
+    generators = "cmake"
+    no_copy_source = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions['BUILD_HTML_DOCS'] = False
+        cmake.definitions['BUILD_QTHELP_DOCS'] = False
+        cmake.definitions['BUILD_MAN_DOCS'] = False
+        cmake.definitions['SHARE_INSTALL_DIR'] = os.path.join(self.package_folder, "res")
+        cmake.configure(source_folder=os.path.join(self.source_folder, "extra-cmake-modules-{}".format(self.version)))
+        return cmake
+
```

Comment:
- cache the cmake object to avoid reconfiguration in package, after build
- rename the `extra-cmake-modules-$VERSION` folder to `source_subfolder` (as is done in all other recipes)
- I don't think this recipe needs to have a `exports_sources = "CMakeLists.txt"` as it is a header-only recipe without a build.

```suggestion
    no_copy_source = True
    
    @property
    def _source_subfolder(self):
        return "source_subfolder"

    _cmake = None

    def source(self):
        tools.get(**self.conan_data["sources"][self.version])
        os.rename("extra-cmake-modules-{}".format(self.version), self._source_subfolder)

    def _configure_cmake(self):
        if self._cmake is not None:
            return self._cmake
        self._cmake = CMake(self)
        self._cmake.definitions["BUILD_HTML_DOCS"] = False
        self._cmake.definitions["BUILD_QTHELP_DOCS"] = False
        self._cmake.definitions["BUILD_MAN_DOCS"] = False
        self._cmake.definitions["SHARE_INSTALL_DIR"] = os.path.join(self.package_folder, "res")
        self._cmake.configure(source_folder=self._source_subfolder)
        return self._cmake
```